cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -201,11 +202,25 @@ func (i *deployCommand) Exec(cmd *cobra.Command, args []string) error {
 			Namespace: i.flags.Project,
 		},
 		Spec: v1alpha1.KogitoAppSpec{
-			Replicas: &i.flags.Replicas,
-			Runtime:  v1alpha1.RuntimeType(i.flags.runtime),
+			KogitoServiceSpec: v1alpha1.KogitoServiceSpec{
+				Replicas: i.flags.Replicas,
+				Envs:     shared.FromStringArrayToEnvs(i.flags.Env),
+				Resources: corev1.ResourceRequirements{
+					Limits:   shared.FromStringArrayToResources(i.flags.Limits),
+					Requests: shared.FromStringArrayToResources(i.flags.Requests),
+				},
+			},
+
+			Runtime: v1alpha1.RuntimeType(i.flags.runtime),
 			Build: &v1alpha1.KogitoAppBuildObject{
+				KogitoServiceSpec: v1alpha1.KogitoServiceSpec{
```

Comment:
I'm not sure if this makes sense. Would be better to not bring this spec to the `Build` attribute.

Reply:
Modified to replace `Env` and `Resources` directly.